### DISCUSSION TITLE
Automatic home directories

### DISCUSSION THREAD

**Discussion by fredriksknese** [filebrowser/filebrowser#2198] [Q&A]
I'm using the latest version from "`curl -fsSL https://raw.githubusercontent.com/filebrowser/get/master/get.sh | bash`" (no --version)
Im starting filebrowser with `filebrowser -a "0.0.0.0" --disable-exec --disable-preview-resize --disable-thumbnails --disable-type-detection-by-header`

I'm trying to understand how new users automatic home directories work. I'm not an expert in Go I'm not sure how to verify this. There is no "--verbose" or "--debug" as a parameter. I have checked "Allow users to signup" and "Auto create user home dir while adding new user".

When I add a user from "User Managment"
`2022/10/30 06:12:26 user: test2, home dir: [/users/test1].`

But when I add a user from "Signup"
`2022/10/30 06:12:26 user: test2, home dir: [/].`

Am I wrong or It is the wrong dir [/] and it should be [/users/test2]?

(I dont want to add a bug report if its suppose to work this way)

**Comment by ramiresviana** [comment]
This looks like an bug and should be fixed.
